### ISSUE TITLE
Terrestrial C cycle should calculate GPP and RA -> NPP

### ISSUE THREAD

**Issue by bpbond** [JGCRI/hector#53]
This was raised by Reviewer 1 of the GMD ms.

**Comment by cahartin** [comment]
@bpbond might be a good task for Alexey

**Comment by kdorheim** [comment]
This falls into the category of it might have already been done 🤷

**Comment by bpbond** [comment]
Nope. We discussed this as an option for @skygering 's summer SULI work but ultimately decided not.

**Comment by bpbond** [comment]
Still a good idea but (1) not critical and (2) not gonna happen for v3. Changing milestone.
